src/catch2/catch_tostring.hpp

```diff
@@ -13,7 +13,7 @@
 #include <cstddef>
 #include <type_traits>
 #include <string>
-#include <string.h>
+#include <cstring>
```

Comment:
This header is probably not needed at all if the char_traits thingy is used, see comment bellow.